packages/mui-material/src/Tabs/Tabs.js

```diff
@@ -357,9 +357,8 @@ const Tabs = React.forwardRef(function Tabs(inProps, ref) {
           if (
             process.env.NODE_ENV !== 'test' &&
             !warnedOnceTabPresent &&
-            tabMeta &&
-            tabMeta.width === 0 &&
-            tabMeta.height === 0
+            tab &&
```

Comment:
This is not implementing the fix mentioned in the PR. Can you update the implementation?

Reply:
What do you mean? Here is link to the implementation, that doesn't cause console error (https://codesandbox.io/s/relaxed-browser-ekxg7m). Suggested condition in the issue is incorrect, because it's breaking case, where tab item has set `display: none`- the warning then does not appear.

Reply:
The solution you have is hardcoded to just check the `style` attribute. There are cases where some classname is used for this behavior. This is why it's better to check the width and height of the element rendered.

If something is not working, is better to just commit the changes and we can take a look together at what needs to be fixed. Without seeing what's breaking it's hard to follow up :)